python/create_mask.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python
+
+"""
+
+; Project     : Coronal Hole Identification via Multi-thermal Emission Reconition Algorithm (CHIMERA)
+;
+; Name        : create_mask
+;
+; Purpose     : Create transparent CH masks from previously stored data
+;
+; Syntax      : create_mask.py path 
+;
+; Inputs      : CHIMERA ch_location.txt file
+;
+; Outputs     : Transparent CH mask
+;
+; Keywords    : path = directory path of file	
+;
+; Example     : ipython> % run -i create_mask.py '/path/to/file/location/'
+;
+; Required Packages    : pip install mahotas
+;
+; History     : v1.0, Written 14-Jun-2018, Tadhg Garton, TCD
+;
+; Contact     : gartont@tcd.ie
+;		info@solarmonitor.org
+
+"""
+
+from datetime import datetime
+import glob
+import sys
+
+import mahotas
+import numpy as np
+import matplotlib.pyplot as plt
+import os.path as op
+
+#=====Check python version=====
+
+print("PYTHON: Starting to create CH masks.")
+if sys.version[0] != '3':
+    raise NameError('=====   Python version 3 must be used for create_mask.py   =====')
+
+#=====Establish pathing=====
+path = ''
+if len(sys.argv) > 1:
+    path = sys.argv[1]
+    exis = op.exists(op.join(path,'meta/'))
+    if exis == 0:
+        raise NameError('===== Path to CH location file does not exist =====')
+
+#=====Find CH location file=====
+print("PYTHON: Finding CHIMERA files.")
+f = glob.glob(op.join(path,"meta/*ch_location*.txt"))
```

Comment:
Careful! I've seen more than one bitten by this! `glob` doesn't return a sorted lists of files. Sort `f` if you depend on it.